sequencer/src/main/java/net/consensys/linea/sequencer/txpoolvalidation/validators/ProfitabilityValidator.java

```diff
@@ -68,6 +73,10 @@ public Optional<String> validateTransaction(
               .getNextBlockBaseFee()
               .orElseThrow(() -> new RuntimeException("We only support a base fee market"));
 
+      final BlockHeader chainHeadHeader = blockchainService.getChainHeadHeader();
+      final LineaPricingUtils.PricingData pricingData =
+          LineaPricingUtils.extractPricingFromExtraData(chainHeadHeader.getExtraData());
+
```

Comment:
I think this is not necessary, since current value are in `LineaProfitabilityConfiguration`, that is updated every time a new block arrives

Reply:
I see what you mean, LineaProfitabilityConfiguration instance has fixedCostWei,  variableCostWei. I'll revise the use of PricingData.

Reply:
Also the parsing is already done in https://github.com/Consensys/linea-sequencer/blob/main/sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataHandler.java so we need to make sure to avoid duplication and in necessary extend what we have to adapt to the new uses

Reply:
> I think this is not necessary, since current value are in `LineaProfitabilityConfiguration`, that is updated every time a new block arrives

Yes, I totally missed where LineaExtraDataPlugin extends AbstractLineaRequiredPlugin and integrates the LineaExtraDataHandler with the plugin lifecycle and listens for new blocks using BesuEvents.addBlockAddedListener to process the extra data in every incoming block.

I'll revise the code to make use of LineaProfitabilityConfiguration instead of LineaPricingUtils.PricingData.